src/dev/build/tasks/os_packages/docker_generator/templates/base/Dockerfile

```diff
@@ -79,44 +104,25 @@ RUN for iter in {1..10}; do \
       {{packageManager}} update  && \
       {{packageManager}} upgrade -y  && \
       {{packageManager}} install -y --no-install-recommends \
-       fontconfig fonts-liberation libnss3 libfontconfig1 ca-certificates curl && \
```

Comment:
libfontconfig1 is already a dependency of fontconfig.  fonts-liberation is superseded by the installed font NotoSansCJK.

@elastic/kibana-reporting-services any idea if I'm missing anything?  I'm pretty sure we're okay, tests are passing and my manual checks lgtm.

Reply:
docs at https://www.elastic.co/guide/en/kibana/current/secure-reporting.html#install-reporting-packages